Razor/Core/OverheadManager.cs

```diff
@@ -37,13 +33,10 @@ public class OverheadMessage
 
     public static class OverheadManager
     {
-        private static ListView _listView;
-
-        public static List<OverheadMessage> OverheadMessages = new List<OverheadMessage>();
-
-        public static void SetControls(ListView listView)
-        {
-            _listView = listView;
+        private static List<OverheadMessage> m_overheadMessages = new List<OverheadMessage>();
```

Comment:
Style-wise, Razor historically used m_Variable for instance members. With a capital letter.

The C# ecosystem however has strongly moved toward adopting the .net core coding style everywhere though and they would have written _variable. I think either choice is fine here, but not m_variable.

Reply:
Yep, I'd like to use the `_variable` style on any new or refactored code.  That can be cleaned up later too.

Reply:
Alrighty, fixed it up into `_variable` where it was touched.
Please feel free to poke at any other style issues, 'cause .net-wise one could say I've been living under a rock for a few years.